Issue #974 Moves help website url to admin user pages

Jake-Bodin opened the pull request:
# Description

To allow non-admins to control OED as much as possible, moves help website url to be a part of the admin user pages. The default value is the standard location of OED help on the web (current value set by code).

Fixes #974 


## Type of change

- [x] Note merging this changes the database configuration.
- [x] This change requires a documentation update

## Checklist

(Note what you have done by placing an "x" instead of the space in the [ ] so it becomes [x]. It is hoped you do all of them.)

- [x] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [x] I have removed text in ( ) from the issue request
- [x] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.github.io/developer/cla.html) and each author is listed in the Description section.

## Limitations

Merge Conflicts - at least 6
Discussed with project lead who decided to handle these in the pull request.

Jake-Bodin committed:
Various changes to add redux state/admin setting

Jake-Bodin committed:
Merge branch 'OpenEnergyDashboard:development' into Issue974

Jake-Bodin committed:
Various changes to titles and additions to server

Jake-Bodin committed:
Merge branch 'Issue974' of https://github.com/Jake-Bodin/OED into Issue974

Jake-Bodin committed:
Changes to server and client parts adding url

Jake-Bodin committed:
Changes adding help_url

Jake-Bodin committed:
Merge branch 'OpenEnergyDashboard:development' into Issue974-branch

Jake-Bodin committed:
Changes to Base_Url, now acquired from state

huss committed:
Merge remote-tracking branch 'origin/development' into pr/Jake-Bodin/1096

huss commented:
There was only one small merge conflict after the latest PR. I merged in development into this PR so that limitation is now resolved.

huss committed:
misc changes

- Made variable names consistent with not being a global constant.
- formatting

huss reviewed (approved):
Thanks to @Jake-Bodin for another contribution. This worked fine and as expected. I made a few miscellaneous changes for names/format that are all minor. It was easier to do than comment. This is all ready to merge.
